### PR TITLE
Quick english suggestions of improvement for the readme

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@
...
 * A built-in list of useful functions:
-    * [`run()`](https://castor.jolicode.com/getting-started/run/#the-run-function): Runs external processes, enabling seamless integration with external tools
-    * [`io()`](https://castor.jolicode.com/going-further/helpers/console-and-io/#the-io-function): Displays beautiful output and interacts with the terminal
-    * [`watch()`](https://castor.jolicode.com/going-further/helpers/watch/): Watches files and automatically triggers actions on file modifications
-    * [`fs()`](https://castor.jolicode.com/going-further/helpers/filesystem/#the-fs-function): Creates, removes, and manipulates files and directories
+    * [`run()`](https://castor.jolicode.com/getting-started/run/#the-run-function): Run external processes, enabling seamless integration with external tools
+    * [`io()`](https://castor.jolicode.com/going-further/helpers/console-and-io/#the-io-function): Display beautiful output and interacts with the terminal
+    * [`watch()`](https://castor.jolicode.com/going-further/helpers/watch/): Watche files and automatically triggers actions on file modifications
```
Comment (ternel) [inline README.md:21]:
```suggestion
    * [`watch()`](https://castor.jolicode.com/going-further/helpers/watch/): Watch files and automatically triggers actions on file modifications
```

Comment (Nek-) [reply README.md]: Yes nice catch sorry ^^ .

**Review by lyrixx — COMMENTED** [review]
Mouais, Not sure it's better. I asked a native speaker and both are valid. 

So i'm +0 on this one. Let's wait more feedback

**Review by pyrech — COMMENTED** [review]
I'm +0.5 on this, I think I prefer this version but without any strong opinion on which one is better

**Comment by lyrixx** [comment]
Thanks

**Pull request merged**